lib/rubocop/cop/chef/correctness/incorrect_library_injection.rb

```diff
@@ -29,10 +29,10 @@ module ChefCorrectness
         #
         #   # good
         #   ::Chef::DSL::Recipe.send(:include, Filebeat::Helpers) # covers previous Recipe & Provider classes
-        #   ::Chef::DSL::Resource.send(:include, Filebeat::Helpers)
+        #   ::Chef::DSL::Resources.send(:include, Filebeat::Helpers)
         #
         class IncorrectLibraryInjection < Cop
-          MSG = 'Libraries should be injected into the Chef::DSL::Recipe or Chef::DSL::Resource classes and not Recipe/Resource/Provider classes directly.'.freeze
+          MSG = 'Libraries should be injected into the Chef::DSL::Recipe or Chef::DSL::Resources classes and not Recipe/Resource/Provider classes directly.'.freeze
```

Comment:
That should be Chef::DSL::Universal, the Chef::DSL::Resources class is marked `@api private` and injecting a library into that class probably just won't work (it lacks the LazyModuleInclude helper that are in the Universal + Recipe DSLs.